docs/Anjay_integrations/Anjay_Nordic_integration/Thingy91.md

```diff
@@ -63,25 +84,10 @@ After you have programmed and reset the board, go to the next step.
     config  :Configure Anjay params
     ```
 
-3. Change the default credentials to your custom data by following the instructions presented in the program and save it.
-
-## Connect to the LwM2M Server
-
-To connect to Coiote IoT Device Management LwM2M Server, please register at https://eu.iot.avsystem.cloud/.
+    !!! tip
+        To show available subcommands, use the **Tab** key.
 
-To connect the board:
-
-1. Log in to Coiote DM and from the left side menu, select **Device Inventory**.
-2. In **Device Inventory**, click **Add device**.
-3. Select the **Connect your LwM2M device directly via the Management server** tile.
-    ![Add via Mgmt](images/mgmt_tile.png "Add via Mgmt")
-4. In the **Device credentials** step:
-     - In the **Device ID** enter your board endpoint name, e.g. `test_device`.
-         ![Device credentials step](images/add_mgmt_quick.png "Device credentials step")
-     - In the **Security mode** section, select the **PSK** mode:
-         - In the **Key identity** field, type `test_device`
-         - In the **Key** field, type the shared secret used in the device-server authentication.
-5. Click the **Add device** button and **Confirm** in the confirmation pop-up.
-6. In the **Connect your device** step, wait for the board to connect.
-7. Click **Go to device** to see your added device dashboard.
+0. Optionally, change the default credentials by following the instructions in the program.
```

Comment:
If i were a first-time user I'd love to see a console screenshot of e.g. renaming the endpoint name to `test_device` or something.